docs/installation/sources/sources_windows.rst

```diff
@@ -0,0 +1,241 @@
+
+.. code-block:: bash
+
+    $ mkdir Fast-DDS-ws && cd Fast-DDS-ws
+    $ wget https://raw.githubusercontent.com/eProsima/Fast-RTPS/master/fastrtps.repos
+    $ mkdir src
+    $ vcs import src < fastrtps.repos
+
+Finally, use colcon_ to compile all software:
+
+.. code-block:: bash
+
+    colcon build
+
+To run an *eProsima Fast DDS* instance, you need to source the colcon overlay with the command:
+
+.. code-block:: bash
+
+    setup.bat
+
+.. important::
+
+    The sourcing of the local colcon overlay is required every time the colcon workspace is opened in a new shell
+    environment to run an *eProsima Fast DDS* instance.
+    As an alternative, you can add it permanently to your :code:`PATH`.
+
+.. note::
+
+    If you want to compile the examples, you will need to add the flag
+    :code:`--cmake-args "-DCOMPILE_EXAMPLES=ON"` when running :code:`colcon build`.
+    If you want to compile the performance tests, you will need to add the flag
+    :code:`--cmake-args "--DPERFORMANCE_TESTS=ON"` when running :code:`colcon build`.
+    For this step, you need Gtest_ as explained in the requirements_windows_sources_ section above.
+
+
+.. _cmake_installation_windows:
+
+CMake Installation
+-------------------
+
+This section explains how to compile *eProsima Fast DDS* locally with CMake_.
+Open a command prompt, and create a :code:`Fast-DDS` directory where to download and build *eProsima Fast DDS* and
+its dependencies:
+
+.. code-block:: bash
+
+    mkdir Fast-DDS && cd Fast-DDS
+
+Now clone the following dependencies and compile them using CMake_.
+
+* `Foonathan memory <https://github.com/foonathan/memory>`_
+
+  .. code-block:: bash
+
+      $ git clone https://github.com/eProsima/foonathan_memory_vendor.git
+      $ cd foonathan_memory_vendor
+      $ mkdir build && cd build
+      $ cmake .. -DCMAKE_INSTALL_PREFIX=../../install
+      $ cmake --build . --target install
+      $ cd ../..
+
+* `Fast CDR <https://github.com/eProsima/Fast-CDR.git>`_
+
+  .. code-block:: bash
+
+      $ git clone https://github.com/eProsima/Fast-CDR.git
+      $ cd Fast-CDR
+      $ mkdir build && cd build
+      $ cmake .. -DCMAKE_INSTALL_PREFIX=../../install
+      $ cmake --build . --target install
+      $ cd ../..
+
+Once all dependencies are installed, you will be able to compile and install *eProsima Fast DDS*:
+
+.. code-block:: bash
+
+    $ git clone https://github.com/eProsima/Fast-RTPS.git
+    $ cd Fast-RTPS
+    $ mkdir build && cd build
+    $ cmake ..  -DCMAKE_INSTALL_PREFIX=../../install -DCMAKE_PREFIX_PATH=../../install
+    $ cmake --build . --target install
+
+
+If you want to install *eProsima Fast DDS* system-wide instead of locally, you need to remove all the flags that
+appear in the configuration steps of :code:`Fast-CDR` and :code:`Fast-RTPS`, and change the one in the
+configuration step of :code:`foonathan_memory_vendor` to the following:
+
+.. code-block:: bash
+
+    -DCMAKE_INSTALL_PREFIX=/usr/local/ -DBUILD_SHARED_LIBS=ON
```

Comment:
Are you sure `/usr/local` is the installation directory in windows?

Reply:
not really. Should check with Barro.

Reply:
Not sure what should be put here.

Reply:
maybe the same path used below?